lib/std/crypto.zig

```diff
@@ -4,8 +4,9 @@
 // The MIT license requires this copyright notice to be included in all copies
 // and substantial portions of the software.
 
-/// Authenticated Encryption with Associated Data
 pub const aead = struct {
+    //! Authenticated Encryption with Associated Data
```

Comment:
We generally prefer `///` comments. The `//!` form was added so that files could document themselves.

Reply:
I know. But auto-doc tool will not show `///` comments for namespaces, and it makes sense, there is only one top level doc comment for a namespace, but it can have multiple aliases each with its own docs, so what should be displayed it that case? I think the top level doc comment. So i converted these to get them displayed in auto-docs, also its more consistent to document namespaces always with `//!`

Reply:
Generated docs not showing doc comments for namespaces is a bug in the generated docs, not with how things are documented. Any aliasing issues would also happen with regular structs since they are the same thing to the compiler.  `//!` comments are mostly intended for top-level use.

Reply:
@Vexu is there any reason to allow `//!` not being actually top level?

Reply:
I don't think so, just more convenient to parse. Further discussion here https://github.com/ziglang/zig/pull/5801#discussion_r450041448.